get_query_generators.php

```diff
@@ -75,7 +75,7 @@ function gen_query_user_role()
  */
 function gen_query_survey_responses()
 {
-    return "SELECT question_id, GROUP_CONCAT(answer) as answers
+    return "SELECT question_id, GROUP_CONCAT(answer SEPARATOR '----') as answers
```

Comment:
I don't like this...you should be returning an array. Don't join just to explode later.